deploy.sh

```diff
@@ -44,9 +44,9 @@ docker push $IMAGE:latest
 echo "Deploying to Google Cloud Run"
 
 gcloud beta run deploy $CLOUD_RUN_SERVICE --image $IMAGE:latest \
---min-instances=1 --memory 4G --cpu=2 --platform managed --no-traffic --tag=test \
+--min-instances=1 --memory 256M --cpu=2 --platform managed --no-traffic --tag=test \
```

Comment:
256M should be much more than needed for a simple flask API. A quick look at the metrics suggests that around 128M would suffice, but that would be close to the limits, so I recommend leaving it like this for now

Reply:
Sounds good!